static/build/.tmp_update/script/network/update_networking.sh

```diff
@@ -580,6 +581,11 @@ log() {
     fi
 }
 
+get_password() {
+    # Get password from file for use with network services authentication
+    export PASS=$(cat "$sysdir/config/.password.txt")
```

Comment:
Better to not export it.

Simply setting it will make it globally available inside this script - we don't want to make it available for every subsequent script (good security practice).